src/server/common/helpers/redis-client.js

```diff
@@ -1,63 +1,44 @@
 import IoRedis from 'ioredis'
-
-import { config } from '~/src/config/index.js'
 import { createLogger } from '~/src/server/common/helpers/logging/logger.js'
 
 /**
  * Setup Redis and provide a redis client
  *
  * Local development - 1 Redis instance
  * Environments - Elasticache / Redis Cluster with username and password
- *
+ * @param {RedisConfig} redisConfig - Redis config
  * @returns {Cluster | Redis}
  */
-function buildRedisClient() {
+function buildRedisClient(redisConfig) {
   const logger = createLogger()
   const port = 6379
   const db = 0
-  const redisConfig = config.get('redis')
   const keyPrefix = redisConfig.keyPrefix
   const host = redisConfig.host
-  let redisClient
 
-  if (redisConfig.useSingleInstanceCache) {
-    redisClient = new IoRedis({
-      port,
-      host,
-      db,
-      keyPrefix
-    })
-  } else {
-    redisClient = new IoRedis.Cluster(
-      [
-        {
-          host,
-          port
-        }
-      ],
-      {
+  const client = redisConfig.useSingleInstanceCache
+    ? new IoRedis({ port, host, db, keyPrefix })
+    : new IoRedis.Cluster([{ host, port }], {
```

Comment:
Using the named imports to match `@returns`

```suggestion
    ? new Redis({ port, host, db, keyPrefix })
    : new Cluster([{ host, port }], {
```